### PR TITLE
i18n(ko-KR): create `upgrade-to/v5.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- create `upgrade-to/v5.mdx`
- #9240 
- #10142 
- #10170
- #10208
- #10209

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by jsparkdev** [comment]
Once #10205, #10224, #10225, #10204 are merged, I expect to be able to merge this PR successfully.

**Review by yanthomasdev — APPROVED** [review]
LGTM! I'll merge the umbrella PR once I approve all of the individual PRs.

**Pull request closed without merging**
